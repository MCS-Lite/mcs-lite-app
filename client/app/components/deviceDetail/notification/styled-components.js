import styled from 'styled-components';

export const Z_INDEX = 999;   // Remind: same as header

export const Fixed = styled.div`
  position: fixed;
  left: 0;
  right: 0;
  margin-top: 10px;
  z-index: ${Z_INDEX};
  width: 100vw;
`;

export const RightWrapper = styled.div`
  display: flex;
  align-items: center;
`;

export const IconWrapper = styled.div`
  font-size: 24px;
  margin-left: 10px;
  cursor: pointer;
`;
