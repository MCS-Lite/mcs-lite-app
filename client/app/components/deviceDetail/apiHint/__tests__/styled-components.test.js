import React from 'react';
import { mount } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import { theme } from 'mcs-lite-theme';
import { CodeWrapper } from '../styled-components';

jest.mock('mcs-lite-ui');

it('should render components correctly', () => {
  const wrapper = mount(
    <ThemeProvider theme={theme}>
      <div>
        <CodeWrapper />
      </div>
    </ThemeProvider>
  );

  expect(wrapper.find('div')).toMatchSnapshot();
});
