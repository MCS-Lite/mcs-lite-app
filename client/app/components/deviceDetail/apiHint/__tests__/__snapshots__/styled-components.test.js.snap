// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render components correctly 1`] = `
Array [
  .c0 {
  max-height: 250px;
  min-height: 3em;
  overflow-y: auto;
  border-radius: 3px;
  margin-top: 10px;
  padding: 5px 10px;
  border: 1px solid #D1D2D3;
  background-color: #FAFAFA;
}

<div>
    <styled-components__CodeWrapper>
        <div
            className="c0"
        />
    </styled-components__CodeWrapper>
</div>,
  .c0 {
  max-height: 250px;
  min-height: 3em;
  overflow-y: auto;
  border-radius: 3px;
  margin-top: 10px;
  padding: 5px 10px;
  border: 1px solid #D1D2D3;
  background-color: #FAFAFA;
}

<div
    className="c0"
/>,
]
`;
